src/main/java/uk/gov/hmcts/reform/sscs/service/VenueService.java

```diff
@@ -24,13 +24,17 @@ public String getEpimsIdForVenue(String processingVenue) {
         String venueId = String.valueOf(airLookupService.getLookupVenueIdByAirVenueName()
             .get(processingVenue));
 
+        return getEpimsIdForVenueId(venueId);
+    }
+
+    public String getEpimsIdForVenueId(String venueId) {
         VenueDetails venueDetails = venueDataLoader.getVenueDetailsMap()
             .get(String.valueOf(venueId));
 
         return Optional.ofNullable(venueDetails)
             .map(VenueDetails::getEpimsId)
             .orElseThrow(() -> {
-                throw new IllegalStateException("Unable to find epims ID for processing venue: " + processingVenue);
+                throw new IllegalStateException("Unable to find epims ID for venue ID: " + venueId);
             });
     }
```

Comment:
I think this method should return the venue, not the epims id, it will make it more useful in other future methods as those methods can get the field they want in that method.

Reply:
as above